adaptations/KawaiK5000.py

```
@@ -0,0 +1,584 @@
+    bank_byte = selected_bank["bank_byte"]
+
+    # Construct the modified SysEx message
+    modified_message = message[:7] + [bank_byte, patch_number] + message[9:]
+
+    return modified_message
+
+
+def numberFromDump(message) -> int:  # where can I see if successful?
+    if isSingleProgramDump(message):
+        return message[8]
+    raise Exception("Can extract number only from single program dump messages")
```

Ruff (0.14.14)

[warning] Create your own exception

(TRY002)